.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/Azure/sonic-swss-common
 [submodule "sonic-linux-kernel"]
 	path = src/sonic-linux-kernel
-	url = https://github.com/Azure/sonic-linux-kernel
+	url = https://github.com/MiTAC-EBU/sonic-linux-kernel.git
```

Comment:
you probably do not want to change this.

Reply:
I have restored the contents of .gitmodules to keep Azure sonic linux kernel and opened a new pull request(https://github.com/Azure/sonic-linux-kernel/pull/39) for my patches.